.github/workflows/coverity.yml

```diff
@@ -24,6 +24,11 @@ jobs:
         export COVERITY_SCAN_BUILD_COMMAND="meson compile -C build"
         export COVERITY_SCAN_TOKEN="${{ secrets.COVERITY_SCAN_TOKEN }}"
 
+        if [ "$COVERITY_SCAN_TOKEN" != "rauc/rauc" ]; then
+            echo "Note: empty COVERITY_SCAN_TOKEN; not running coverity"
+            exit
+        fi
```

Comment:
This doesn't check if the token is empty, but if it differs from "rauc/rauc".

Reply:
That is correct… my first try was testing for COVERITY_SCAN_PROJECT_NAME instead but then I decided that the token makes more sense… 🙈 Fixed it, and improved the code flow a bit too.

Reply:
Hmm, I'd definitely prefer an early exit to indenting the normal case further.

Reply:
OK.